### ISSUE TITLE
after make install success, failed do sudo make uninstall

### ISSUE THREAD

**Issue by iysheng** [mono/mono#21818]
## Steps to Reproduce

1.  sudo make install
2.  sudo make uninstall

### Current Behavior

MONO_PATH="./../../class/lib/build:$MONO_PATH" /home/yjoy/gProjects/mono/runtime/mono-wrapper  ./../../class/lib/build/gensources.exe --strict --platformsdir:./../../build "../../build/deps/linux_net_4_x__corlib.dll.sources" "corlib.dll" "linux" "net_4_x"
The assembly mscorlib.dll was not found or could not be loaded.
It should have been installed in the `/usr/local/lib/mono/4.5/mscorlib.dll' directory.
rm -f /usr/local/lib/mono/4.5/mscorlib.dll /usr/local/lib/mono/4.5/mscorlib.dll.mdb /usr/local/lib/mono/4.5/mscorlib.pdb
make[7]: Leaving directory '/home/yjoy/gProjects/mono/mcs/class/corlib'
make[6]: Leaving directory '/home/yjoy/gProjects/mono/mcs/class/corlib'
make[6]: Entering directory '/home/yjoy/gProjects/mono/mcs/class/Mono.Security'
MONO_PATH="./../../class/lib/build:$MONO_PATH" /home/yjoy/gProjects/mono/runtime/mono-wrapper  ./../../class/lib/build/gensources.exe --basedir:./Test --strict --platformsdir:./../../build "../../build/deps/linux_net_4_x_Mono.Security_test.dll.response" "Mono.Security_test.dll" "linux" "net_4_x"
The assembly mscorlib.dll was not found or could not be loaded.
It should have been installed in the `/usr/local/lib/mono/4.5/mscorlib.dll' directory.
MONO_PATH="./../../class/lib/build:$MONO_PATH" /home/yjoy/gProjects/mono/runtime/mono-wrapper  ./../../class/lib/build/gensources.exe --strict --platformsdir:./../../build "../../build/deps/linux_net_4_x__Mono.Security.dll.sources" "Mono.Security.dll" "linux" "net_4_x"
The assembly mscorlib.dll was not found or could not be loaded.
It should have been installed in the `/usr/local/lib/mono/4.5/mscorlib.dll' directory.
make uninstall-local
make[7]: Entering directory '/home/yjoy/gProjects/mono/mcs/class/Mono.Security'
MONO_PATH="./../../class/lib/build:$MONO_PATH" /home/yjoy/gProjects/mono/runtime/mono-wrapper  ./../../class/lib/build/gensources.exe --basedir:./Test --strict --platformsdir:./../../build "../../build/deps/linux_net_4_x_Mono.Security_test.dll.response" "Mono.Security_test.dll" "linux" "net_4_x"
The assembly mscorlib.dll was not found or could not be loaded.
It should have been installed in the `/usr/local/lib/mono/4.5/mscorlib.dll' directory.
MONO_PATH="./../../class/lib/build:$MONO_PATH" /home/yjoy/gProjects/mono/runtime/mono-wrapper  ./../../class/lib/build/gensources.exe --strict --platformsdir:./../../build "../../build/deps/linux_net_4_x__Mono.Security.dll.sources" "Mono.Security.dll" "linux" "net_4_x"
The assembly mscorlib.dll was not found or could not be loaded.
It should have been installed in the `/usr/local/lib/mono/4.5/mscorlib.dll' directory.
make[7]: *** No rule to make target '../../class/lib/build/gacutil.exe', needed by 'uninstall-local'.  Stop.
make[7]: Leaving directory '/home/yjoy/gProjects/mono/mcs/class/Mono.Security'
make[6]: *** [../../build/rules.make:220: do-uninstall] Error 2
make[6]: Leaving directory '/home/yjoy/gProjects/mono/mcs/class/Mono.Security'
make[5]: *** [../build/rules.make:239: uninstall-recursive] Error 1
make[5]: Leaving directory '/home/yjoy/gProjects/mono/mcs/class'
make[4]: *** [build/rules.make:239: uninstall-recursive] Error 1
make[4]: Leaving directory '/home/yjoy/gProjects/mono/mcs'
make[3]: *** [Makefile:57: profile-do--net_4_x--uninstall] Error 2
make[3]: Leaving directory '/home/yjoy/gProjects/mono/mcs'
make[2]: *** [Makefile:53: profiles-do--uninstall] Error 2
make[2]: Leaving directory '/home/yjoy/gProjects/mono/mcs'
make[1]: *** [Makefile:724: uninstall-mcs] Error 2
make[1]: Leaving directory '/home/yjoy/gProjects/mono/runtime'
make: *** [Makefile:602: uninstall-recursive] Error 1

## On which platforms did you notice this

[ ] macOS
[x] Linux
[ ] Windows

**Version Used**:

commit 0f53e9e151d92944cacab3e24ac359410c606df6

## Stacktrace

```
Please paste the stack trace here if available.
```

**Comment by lextm** [comment]
Please read #21796 and learn where to find the latest source code by the new maintainers. This repo is deprecated.

**Comment by iysheng** [comment]
yeah I do ``sudo make uninstall`` in gitlab repo [mono](https://gitlab.winehq.org/mono/mono) success. tks.
